spec/formatting.md

```diff
@@ -8,6 +8,16 @@ when formatting a message for display in a user interface, or for some later pro
 The document is part of the MessageFormat 2.0 specification,
 the successor to ICU MessageFormat, henceforth called ICU MessageFormat 1.0.
 
+## Literal Resolution
+
+The resolved value of _text_, _literal_ and _nmtoken_ tokens
+is always a string concatenation of its parts,
```

Comment:
What are the "parts" here? I would think these items (_text_, _literal_ and _nmtoken_) are part-less?

Reply:
This is meant to refer to the _*-char_ and _*-escape_ parts of _text_ and _literal_, and _name-char_ for _nmtoken_, as hinted by the rest of this sentence.